src/html/modals/error.pug

```diff
@@ -23,15 +23,15 @@
   script.
     ipcRenderer.on("user:error", (event, error, restart, ignore) => {
       if(global.packageInfo.updateAvailable) {
-        $("#not-latest-stable").show();
+        document.getElementById("not-latest-stable").style.display = "block";
```

Comment:
It'd be better to create a new hide function to make sure this is agnostic of what block the element is supposed to have.

new css class:

```css
.hidden {
  display: none !important;
}
```

and then the management functions:

```javascript
const visibility = {
  show: (id) => {
    document.getElementById(id).classList.remove("hidden");
  },
  hide: (id) => {
    document.getElementById(id).classList.add("hidden");
  },
  toggle: (id) => {
    document.getElementById(id).classList.toggle("hidden");
  }
}
```

Reply:
I usually tend to disagree with the use of  **!important** and css manipulation from the sake of altering the DOM. If you ever need to add another property that uses display you always have to check to see if the class is present or otherwise the specificity could mess up and that is cumbersome.

I totally agree with the functions though;

```
const visibility = {
  show: (id) => {
    document.getElementById(id).style.display = "block"
  },
  hide: (id) => {
    document.getElementById(id).style.display = "none"
  },
  toggle: (id) => {
  const element =   document.getElementById(id)
   element.style.display = element.style.display === "block" ? "none" : "block";
  }
}

```
What are your thoughts in relation to that approach ?

Reply:
This is the singular intended usecase of `!important`. You're absolutely right, it should never be used out of laziness and not wanting to deal with the cascade, but for classes that are supposed to change the behavior everywhere. The advantage is that these functions would work on elements with other `display` (eg. `inline`, `float`) as well.